support/investigating-pod-issues.adoc

```diff
@@ -0,0 +1,30 @@
+[id="investigating-pod-issues"]
+= Investigating pod issues
+include::modules/common-attributes.adoc[]
+:context: investigating-pod-issues
+
+toc::[]
+
+{product-title} {product-version} leverages the Kubernetes concept of a pod, which is one or more containers deployed together on one host. A pod is the smallest compute unit that can be defined, deployed and managed on {product-title} {product-version}.
+
+After a pod is defined, it is assigned to run on a node until its container (or containers) exit, or until it is removed. Depending on policy and exit code, pods are either removed after exiting or retained so that their logs can be accessed.
```

Comment:
We try to avoid parentheses, so just `container or containers exit`

Reply:
Understood, I will keep this in mind in my writing. Resolved.